gen/templates/resource_example_all_attributes.tf.tmpl

```diff
@@ -18,9 +18,7 @@ resource "aci_{{$.ResourceName}}" "full_example_{{getResourceName $key $.Definit
       {{- end}}
     {{- end}}
     {{- else if .IgnoreInTest}}
-  {{overwriteProperty .PkgName .SnakeCaseName $.Definitions}} = "{{.IgnoreInTestExampleValue}}"
-    {{- else if eq .ValueType "bitmask"}}{{ $bitmaskTestValue := lookupTestValue .PkgName .SnakeCaseName $.TestVars $.Definitions $.Properties}}
-  {{overwriteProperty .PkgName .SnakeCaseName $.Definitions}} = [{{range $index, $value := $bitmaskTestValue}}{{if lt $index (subtract (len $bitmaskTestValue) 1)}}"{{$value}}",{{else}}"{{$value}}"{{end}}{{end}}]
+  {{overwriteProperty .PkgName .SnakeCaseName $.Definitions}} = {{.IgnoreInTestExampleValue}}
```

Comment:
Can you add this back?
```
{{- else if eq .ValueType "bitmask"}}{{ $bitmaskTestValue := lookupTestValue .PkgName .SnakeCaseName $.TestVars $.Definitions $.Properties}}
  {{overwriteProperty .PkgName .SnakeCaseName $.Definitions}} = [{{range $index, $value := $bitmaskTestValue}}{{if lt $index (subtract (len $bitmaskTestValue) 1)}}"{{$value}}",{{else}}"{{$value}}"{{end}}{{end}}]
```

Reply:
I move this logic to the [formatSlice](https://github.com/CiscoDevNet/terraform-provider-aci/pull/1294/files#diff-b6ad5742505b641f960dc7f467e0720e4b92b11a338d75afaf02aa4cc9edcc08R1828) function in the generator instead. I preferred sharing the same logic to format slices in one place. This also reduces complexity in the template.

Reply:
Ah I see that you changed it in LookupTestValue. Cool